FULL_HELP_DOCS.md

```diff
@@ -1555,6 +1556,49 @@ Sign, Simulate, and Send transactions
 
 
 
+## `stellar tx update`
+
+Update the transaction
+
+**Usage:** `stellar tx update <COMMAND>`
```

Comment:
I decided to put this command under `stellar tx update` instead of `stellar tx edit`, since it does feel like a different set of commands than the current edit command, that opens an editor and allows the users to edit the tx json.